lib/src/arithmetic/floating_point/fft/butterfly.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
please update headers on the files to match  the recommended style, including the name of the file, date authored, purpose of the file, etc.